app/views/cohorts/show/_form.html.haml

```diff
@@ -0,0 +1,44 @@
+%a{ name: "register" }
+%section.contributing-section.pb-80
+  .lg-container.flex.flex-col.items-center
+    %h2.text-h4.mb-16.flex.items-center
+      = graphical_icon "concepts"
+      .text-gradient.ml-24 Get Registered!
+
+    - if !user_signed_in?
+      %h3.text-h1.text-center.mb-16 Sign up to the #{cohort.name}
+      %p.text-p-xlarge.mb-16.text-center.mx-auto{ style: "max-width:600px;" }
+        To register for the #{cohort.name} please log in, or sign up to Exercism, and then come back to this page.
+
+      .buttons.flex.mb-40.flex-col.sm:flex-row
+        = link_to "Sign up for free", new_user_registration_path, class: 'btn-primary btn-l mb-12 sm:mb-0 sm:mr-16'
+        = link_to "Log in", new_user_session_path, class: 'btn-secondary btn-l shadow-buttonS'
+
+    - elsif membership
+      - if membership.enrolled?
+        %h3.text-h1.text-center.mb-16 You're all set!
+        %p.text-p-xlarge.mb-16.text-center.mx-auto{ style: "max-width:800px;" }
+          Congratulations - you're all set!
+          One of our team will email you shortly to welcome you to the #{cohort.name} and tell you how to get set up!
+      - else
+        %h3.text-h1.text-center.mb-16 You're on the waiting list
+        %p.text-p-xlarge.mb-16.text-center.mx-auto{ style: "max-width:800px;" }
+          Sorry - the #{cohort.name} filled up before you got here! You're currently on the waiting list. If a space becomes available we'll let you know. Otherwise we'll message you about the next #{cohort.name} later in the year.
```

Comment:
One thing that we lost with this PR is the bit about the position in the waiting list. I could add that back if we want to, but then I'd need to store the enrollment position with the cohort membership 🤷